xscen/io.py

```diff
@@ -386,7 +386,7 @@ def save_to_zarr(
 
     Returns
     -------
-    None
+    dask.delayed object if compute=False, None otherwise.
```

Comment:
Just curious, what's the use case of doing a `save_to_zarr` with `compute=False`?

Reply:
Saving to datasets at the same time! I had a use for it with the diagnostics : I don't want to redo the same computation twice, so I want the saving of the properties and of the measures to be done at the same time.